docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb

```diff
@@ -2,19 +2,24 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    titiler_endpoint = "https://titiler.ops.maap-project.org"
</pre><p>We might want to swap titiler endpoints - titiler-stac.maap-project.org for now?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>